plugins/pulp_rpm/plugins/importers/yum/sync.py

```diff
@@ -198,13 +198,15 @@ def get_metadata(self):
         try:
             metadata_files.download_repomd()
         except IOError, e:
-            raise FailedException(str(e))
+            raise PulpCodedException(error_code=error_codes.RPM1004, reason=str(e))
```

Comment:
This makes sense that failure to parse would log loudly and failure to download would not. Let's leave it as is. Good job looking into this. Thanks.